rsbuild.config.ts

```diff
@@ -72,7 +72,7 @@ const appConfig = defineConfig({
             'process.env.RELEASE_LINK': JSON.stringify(releaseLink),
             'process.env.RELEASE_CHANGELOG': JSON.stringify(releaseChangelog),
             'process.env.DISABLE_SERVICE_WORKER': JSON.stringify(disableServiceWorker),
-            'process.env.INLINED_APP_CONFIG': JSON.stringify(configJson),
+            'process.env.INLINED_APP_CONFIG': JSON.stringify(configJson.configSource === 'bundled' ? configJson : null),
```

Comment:
Defining this value in the same config which one wants to switch the behaviour of seems like a roundabout way. (As you would need to manually modify the config before building)

I would expect it to be toggleable at built time without modifying the checked out source e.g. via a build param, feature flag, custom goal, or environment variable.

Reply:
Yes, you are absolutely right, it affects only bundling process and should work like in the same way as disabling service worker works

Reply:
Though I put it here to make it noticeable. Imagine you don't know about the build flag, anyway, I'm not sure what behaviour I should use in repo by default (remote I guess?)

Reply:
Well remote would make more sense to keep the behaviour compatible with the previous one.

Reply:
The previous behavior didn't make app menu delay to load before config.json is set, that's why I'm so unsure